src/pages/jobdetail/Apply.js

```diff
@@ -0,0 +1,92 @@
+import styled from 'styled-components';
+import { FaHeart } from 'react-icons/fa';
+
+const Apply = ({ setApply, detailList, setDetailList }) => {
+  const goToApplyInformation = () => {
+    setApply(false);
+  };
+
+  const likeBtn = id => {
+    const newLikeBtn = detailList.map(item => {
+      if (item.id === id) {
+        item.liked = !item.liked;
+        return item;
+      }
+      return item;
+    });
+    setDetailList(newLikeBtn);
+  };
+  return (
+    <ApplyArea>
+      <ApplyTitle>지원하기</ApplyTitle>
+      <ApplyDescription>프론트/백엔드 모바일웹 분야 개발자</ApplyDescription>
+      <ApplyBtn
+        bg="white"
+        color="#3366ff"
+        onClick={() => likeBtn(detailList[0].id)}
+      >
+        좋아요
+      </ApplyBtn>
+      <ApplyBtn bg="#3366ff" color="white" onClick={goToApplyInformation}>
+        지원하기
+      </ApplyBtn>
+      <LikeNum onClick={() => likeBtn(detailList[0].id)}>
+        <FaHeart
+          style={{
+            color: detailList[0]?.liked ? 'red' : '#999999',
+          }}
+        />{' '}
+        <span>{detailList[0]?.liked ? 1 : 0}</span>
```

Comment:
- 인라인 스타일은 최대한 지양해주세요. 스타일드 컴포넌트는 인라인 스타일을 사용하지 않아도 동적으로 구현할 수 있습니다. 
- `{' '}` 필요한 코드인가요?
```suggestion
        <LikeButton isLiked={detailList[0]?.liked} />
        <span>{detailList[0]?.liked ? 1 : 0}</span>
```
```jsx
const LikeButton = styled(FaHeart)`
  margin-right: 5px;
  color: ${({ isLiked }) => (isLiked ? 'red' : '#999999')};
`;
```
이렇게 바꿔볼 수 있을 것 같아요.
styled(컴포넌트)``; 로 ()안의 컴포넌트의 스타일을 확장해서 사용할 수 있습니다.

Reply:
이렇게 하면, 위에서 네스팅해서 span 에 스타일을 준 부분은 불필요합니다.